src/ontology/mondo-edit.obo

```diff
@@ -266388,7 +266379,6 @@ xref: Orphanet:238744 {source="MONDO:equivalentTo", source="OMIM:613689"}
 xref: SCTID:718679004 {source="MONDO:equivalentTo"}
 xref: UMLS:C3150946 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:613689"}
 is_a: MONDO:0002263 {source="MONDO:Redundant", source="Orphanet:238744"} ! female reproductive system disorder
-is_a: MONDO:0015852 {source="Orphanet:238744"} ! excess breast volume or number
 is_a: MONDO:0018234 ! dysostosis
```

Comment:
this source is not correctly formatted.

Reply:
same as above, it will be addressed at some point